app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
```

Comment:
Code Style: Use one style of quotes in your code. Double quotes are preferable. Consider using double quotes consistently throughout the code.